index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Racha's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link 
+      href="https://fonts.googleapis.com/css?family=Poppins:100,200,300,400,500,700"  
+      rel="stylesheet">
+</head>
+<body>
+    <header >
+        <nav class="nav-bar">
+            <ul>
+              <li>
+                <a class="logo" href="index.html">Racha</a>
+              </li>
+            </ul>
+            <ul class="links-holder">
+              <li class="link"><a href="index.html">Portoflio</a></li>
+              <li class="link"><a href="#">About</a></li>
+              <li class="link"><a href="#">Contact</a></li>
+            </ul>
```

Comment:
- [Optional] Kindly consider using an indentation of two spaces in all your documents. This is a best practice and also a convention.